### DISCUSSION TITLE
Unable to handle secondary assignees in Issue API

### DISCUSSION THREAD

**Discussion by RameshwarShindeCctech** [iTwin/itwinjs-core#5306] [Q&A]
Hi, 

I have implemented the create issue API from the [sample](https://www.itwinjs.org/sandboxes/iTwinPlatform/Issues). Able to create issue (POST) with other parameters in the request body, but the API calls are failing for the secondary assignees. 

I have created data similar to what is shown in [sample requests](https://developer.bentley.com/apis/issues-v1/operations/create-issue/#secondary-assignee)

Can I get the help please to get the explanation how to handle secondary assignees?

**Comment by pratikshan85** [comment]
@RameshwarShindeCctech Can you please give me the JSON data during the issue creation which causes the issue?

**Reply by pratikshan85** [reply]
@RameshwarShindeCctech 
Hello,

This API call failed because you don't have the Assign permission for the form you're creating, so you're not allowed to set assignee or assignees.  

Now, to get permission for setting the assignee or assignees you need to have the 
**"Forms_AssignAccess"** permission.

Please refer to the below link for more details about the Assign permission (Please check the permission header).

https://developer.bentley.com/apis/issues-v1/operations/modify-issue/


To set the Permission for a member, please visit the below link.

https://developer.bentley.com/apis/access-control/operations/create-itwin-role/

Please let me know of any further details.
